docs/advanced/api_examples.yml

```diff
@@ -292,6 +292,39 @@
 
 -
   id:
+    "get actual containers"
+
+  doc: |
+    Get a list of all containers that are present in the cluster.
+
+  requires:
+    - "create container"
```

Comment:
Also `create container with restart policy` it looks like?  Since there are two containers in the response.